li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaFederatedConsumerImpl.java

```diff
@@ -113,6 +116,12 @@ public ClusterDescriptor getCluster() {
 
   private int _nextClusterIndexToPoll;
 
+  // If client is in federated mode
+  private boolean _isFederatedMode;
```

Comment:
as federated mode would be the common, expected, state, i'd name this after the less obvious possibility - isFallbackMode, with a javadoc explaining roughly what that means